api/v1/cluster_types.go

```diff
@@ -2280,6 +2280,14 @@ func (cluster *Cluster) SetInheritedDataAndOwnership(obj *metav1.ObjectMeta) {
 	utils.SetOperatorVersion(obj, versions.Version)
 }
 
+// ShouldForceLegacyBackup if present takes a backup without passing the name argument even on barman version 3.3.0+.
+// This is needed to test both backup system in the E2E suite
+func (cluster *Cluster) ShouldForceLegacyBackup() bool {
+	const legacyBackupAnnotationName = "cnpg.io/forceLegacyBackup"
+
+	return cluster.Annotations[legacyBackupAnnotationName] == "true"
+}
+
```

Comment:
never called outside of the E2E... so the E2E will respect the "legacy" Annotation, but the operator ignores it